saleor/graphql/product/resolvers.py

```diff
@@ -16,6 +16,7 @@
     filter_products_by_stock_availability,
     sort_qs,
 )
+from .types import Category, Collection
```

Comment:
Please keep them lazy. Just as it was before, i.e. `"Category"`. This prevents infinite loops when we want to invoke a product resolver from another type's resolver.